core/src/sample_performance_service.rs

```diff
@@ -21,10 +21,9 @@ pub struct SamplePerformanceService {
 impl SamplePerformanceService {
     pub fn new(
         bank_forks: &Arc<RwLock<BankForks>>,
-        blockstore: &Arc<Blockstore>,
+        blockstore: Arc<Blockstore>,
         exit: Arc<AtomicBool>,
     ) -> Self {
-        let blockstore = blockstore.clone();
         let bank_forks = bank_forks.clone();
```

Comment:
this could also be passed as owned

Reply:
Yeah, I noticed that one too; decided to keep this one limited to blockstores and one of us can take another pass on all the `&Arc<BankForks>` as I'm sure this isn't the only one